openhands-sdk/openhands/sdk/conversation/impl/remote_conversation.py

```diff
@@ -666,10 +666,9 @@ def close(self) -> None:
 
         self._end_observability_span()
 
-        try:
-            self._client.close()
-        except Exception:
-            pass
+        # NOTE: We don't close self._client here because it's shared with the workspace.
+        # The workspace owns the client and will close it during its own cleanup.
+        # Closing it here would prevent the workspace from making cleanup API calls.
```

Comment:
Do we need this comment? It seems maybe unnecessary


```suggestion
```

Reply:
I do hope we add it here so someone won't be adding the same thing in the future :(